### PR TITLE
 RORO-4125 [CG] Change NCTS 5 Endpoint

### PR DESCRIPTION
**Pull request by spifix-hmrc**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # microservice specific routes
 
-POST        /ctc/registerncf/v1                                 uk.gov.hmrc.registerncfstub.controllers.RegisterNcfController.receiveNcfData
-POST        /goods-vehicle-movement-system/gb/registerncf/v1    uk.gov.hmrc.registerncfstub.controllers.RegisterNcfController.receiveNcfData
-POST        /ctc/registerncfni/v1                               uk.gov.hmrc.registerncfstub.controllers.RegisterNcfController.receiveNcfData
+POST        /ctc/registerncf/v1                                                 uk.gov.hmrc.registerncfstub.controllers.RegisterNcfController.receiveNcfData
+POST        /goods-vehicle-movement-system/ncts5/noticeofcrossingfrontiergb/v1  uk.gov.hmrc.registerncfstub.controllers.RegisterNcfController.receiveNcfData
```
Comment (tipperba) [inline conf/app.routes]: again, just checking if the path for NCF GB should start with /goods-vehicle-movement-system

Comment (spifix-hmrc) [reply conf/app.routes]: not sure, I see that GB always did and NI not...

Comment (spifix-hmrc) [reply conf/app.routes]: Is this anything to do with scrubbing centre?

Comment (spifix-hmrc) [reply conf/app.routes]: I guess not, have removed

**Review by tipperba — APPROVED** [review]

**Pull request merged**